assets/src/hooks/useNotificationsReducer.ts

```diff
@@ -58,17 +72,17 @@ export const markAllAsRead = (): MarkAllAsReadAction => ({
 interface SetNotificationsAction {
   type: "SET_NOTIFICATIONS"
   payload: {
-    notificationsData: NotificationData[]
+    notifications: Notification[]
```

Comment:
**praise:** thank you for making it consistent whether the actions for this reducer take `Notification`s or `NotificationData`.